utils/boxbackup/Makefile

```diff
@@ -0,0 +1,117 @@
+#
+# Copyright (C) 2017 Val Kulkov <val.kulkov@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=boxbackup
+PKG_SOURCE_DATE:=2017-10-18
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/boxbackup/boxbackup.git
+PKG_SOURCE_VERSION:=24aca3fc618e36e2feb448bd7b5c05b31a064bd3
+PKG_MIRROR_HASH:=ba6a76040c070b864f4d0057c160492eca522107dfa2e006df41abd1554b2161
+PKG_MAINTAINER:=Val Kulkov <val.kulkov@gmail.com>
+
+PKG_LICENSE:=GPL-2.0 BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE.txt LICENSE-DUAL.txt LICENSE-GPL.txt COPYING.txt
+
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/boxbackup/Default
+  SECTION:=utils
+  CATEGORY:=Utilities
+  URL:=https://www.boxbackup.org/
+  TITLE:=A client-server online backup system
+endef
+
+define Package/boxbackup/description/Default
+Box Backup is an open source, completely automatic, on-line backup
+system that supports both continuous and snapshot backups and secure
+file transfer over an untrusted network.
+endef
+
+define Package/boxbackup-server
+  $(call Package/boxbackup/Default)
+  TITLE+= (server)
+  USERID:=bbstored:bbstored
+  DEPENDS:=+libopenssl +zlib +libstdcpp +coreutils-stat +openssl-util
```

Comment:
does this work with uclibc++?

Reply:
I tested this. It does not. I opened an issue upstream: https://github.com/boxbackup/boxbackup/issues/35

Reply:
I have not yet had a chance to look into this issue, but I do plan to look into this at my earliest opportunity.